bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleRegistryImpl.java

```diff
@@ -504,8 +504,8 @@ public void added(Provider<Rule> provider, Rule element) {
             String rUID = ruleEngine.getUniqueId();
             ruleWithUID = initRuleId(rUID, element);
         }
-        super.added(provider, ruleWithUID);
```

Comment:
It will not be enough, because the status events are fired when the rule is added into RuleEngine, which is done in the onAddElement() method at the moment. The logic should be moved from onXXXElement() to  notifyListenersAboutXXXXElement(). I'll try to fix this.
BTW wdyt about removing redundant NOT_INITIALIZED event.
Up to now we have:
added -> not_initillized -> idle
the proposal is:
added -> idle 
or 
added -> not_initilized.